plugins/.gitignore

```diff
@@ -0,0 +1,11 @@
+
+# Ignore Plugin Reserved files in all Plugins.
```

Comment:
I think this file is a good idea, however it ignores more files, than needed. IMO we should not limit it that much.

The following files are reserved by UCRM and should always be ignored:
```
ucrm.json
.ucrm-plugin-running
.ucrm-plugin-execution-requested
```

I would also keep the ignore for `vendor` directory, but that's all.